ElectronyatShop/Dockerfile

```
@@ -5,16 +5,14 @@ EXPOSE 8080
 
 FROM mcr.microsoft.com/dotnet/sdk:9.0 AS build
 ARG BUILD_CONFIGURATION=Release
-WORKDIR /src
-COPY ["ElectronyatShop/ElectronyatShop.csproj", "ElectronyatShop/"]
-RUN dotnet restore "ElectronyatShop/ElectronyatShop.csproj"
+WORKDIR /ElectronyatShop/src
 COPY . .
-WORKDIR "/src/ElectronyatShop"
-RUN dotnet build "./ElectronyatShop.csproj" -c $BUILD_CONFIGURATION -o /ElectronyatShop/build
+RUN dotnet restore
+RUN dotnet build -c $BUILD_CONFIGURATION -o /ElectronyatShop/build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)